### PR TITLE
BFD-3343: Offboard Ad Hoc team

### PR DESCRIPTION
**Pull request by dondevun**
**JIRA Ticket:**
[BFD-3343](https://jira.cms.gov/browse/BFD-3343)

**User Story or Bug Summary:**

---

### What Does This PR Do?

Offboarding Ad-Hoc team

### What Should Reviewers Watch For?

If you're reviewing this PR, please check for these things in particular:

* Verify all PR security questions and checklists have been completed and addressed.

### What Security Implications Does This PR Have?

Submitters should complete the following questionnaire:

* If the answer to any of the questions below is **Yes**, then **you must** supply a link to the associated Security Impact Assessment (SIA), security checklist, or other similar document in Confluence here: **N/A**

    * Does this PR add any new software dependencies? 
      * [ ] Yes
      * [x] No
    * Does this PR modify or invalidate any of our security controls?
      * [ ] Yes
      * [x] No
    * Does this PR store or transmit data that was not stored or transmitted before?
      * [ ] Yes
      * [x] No

* If the answer to any of the questions below is **Yes**, then please add @StewGoin as a reviewer, and note that this PR **should not be merged** unless/until he also approves it.
    * Do you think this PR requires additional review of its security implications for other reasons?
      * [ ] Yes
      * [x] No

### What Needs to Be Merged and Deployed Before this PR?

This PR cannot be either merged or deployed until the following prerequisite changes have been fully deployed:

* N/A

### Submitter Checklist

I have gone through and verified that...:

* [x] I have named this PR and branch so they are [automatically linked](https://confluence.atlassian.com/adminjiracloud/integrating-with-development-tools-776636216.html) to the (most) relevant Jira issue. Ie: `BFD-123: Adds foo`
* [x] This PR is reasonably limited in scope, to help ensure that:
    1. It doesn't unnecessarily tie a bunch of disparate features, fixes, refactorings, etc. together.
    2. There isn't too much of a burden on reviewers.
    3. Any problems it causes have a small "blast radius".
    4. It'll be easier to rollback if that becomes necessary.
* [x] This PR includes any required documentation changes, including `README` updates and changelog / release notes entries.
* [x] The data dictionary has been updated with any field mapping changes, if any were made.
* [x] All new and modified code is appropriately commented, such that the what and why of its design would be reasonably clear to engineers, preferably ones unfamiliar with the project.
* [x] All tech debt and/or shortcomings introduced by this PR are detailed in `TODO` and/or `FIXME` comments, which include a JIRA ticket ID for any items that require urgent attention.
* [x] Reviews are requested from both:
    * At least two other engineers on this project, at least one of whom is a senior engineer or owns the relevant component(s) here.
    * Any relevant engineers on other projects (e.g. DC GEO, BB2, etc.).
* [x] Any deviations from the other policies in the [DASG Engineering Standards](https://github.com/CMSgov/cms-oeda-dasg/blob/master/policies/engineering_standards.md) are specifically called out in this PR, above.
    * Please review the standards every few months to ensure you're familiar with them.

### CODE REVIEW

**Review by sb-benohe — APPROVED** [review]

**Comment by dondevun** [comment]
@sb-benohe

File: ops/terraform/env/mgmt/base_config/values/mgmt.yaml [hunk 1]
```diff
@@ -56,6 +50,6 @@ bfd:
...
...cX2oX6vpVg54Fi2DqSV0ECaaBrz1bk5Wkf5aHf-J97Hnu7s3OYAsZ9ElSsfA-RHe8CP9JCKavbkLvsqNAe3IEK6fC-TR26Y568Jnh1IZJy68TmVdjs9YG-jzg1ks3K6xq_-I-5mn_q5QmteGTkt-vHKD3W0TB2UZjtTVEn6fUamehN5K1cSL-bmcJPLNgLpgkV6YZyUQeBNZ-UWkh3hALVlacGelc2gV51zKd0Wm3MSy5u5eVCDRnfHVw5DEEf8uVSW53Ii0P-NSdcmJIE0W2HBhV1n-bz0rKJ7YTxQHCZH3yjwRF1wMZl-bzrflv0qzIRlLlplfJ3dFzlAteWy_KPvgYfivTY8GnPBC_ffmxf-MfI_PdsNH_Dm7Z31AvUm2ZG8vw-OARhXeplAXtf61O0NUKgmPy48Tl7m_gZUFVtsvHLvJIXMRpCOOiugP1Ica89rNh36rWK1llOewqyaS_pOU_Ta1DU8RPaq49MajEvI1Jm2hneAPpiaH6EoGqXzjjuFRN8EdpIi3cX9ImwDzoh46oBJD_8JqjCwrvyo0ub2luOCjArFuk4d1_WCKo0uajg8kdxOfZl72vv7UOrtivq2uRqxSL0JTk3dHlrOnkEKmr8O2Txw2wqGuQHc5lqIl_3JLZM_5_ohEI316lBLWjSpuY7HF8h0Njqxkrf6f9tCRqpj4H0_mcjeDVfj1rz07OihjTRrSieF3mcvUf5hasMwWjhPgAxPSFCxZZUfE5eT297wRTeBqYj-C1B2hZk2ZjePGDNiUdJMH_JBa5KBWFcHAOg8FOHgj-xqywFJma-sJUy2j9iXNXrrEMYz-THm3p7aJQ1nuEQjAS-VG3X0M0VaGAPdrJgR_mh5bGZ1GZ19g0o1_5smeWhWmvSNiXdBljY0NfWbpu9fnTwJFvvWie0QCYXM2IsC8djId4rYWI2gHFeJsIBv7bfh2U0nJfU4xNGv5RwP_E0qzVFlsmiPSSwJHkBtOg7L7yi2dtvUH7d9p5UEd2KUM7gD5u23cqhYXnDKwWmsx0lfZCx_hBgi7Q7OIAZzBlAjEAhVOUyTdxVvpcqqhILhcDcRDhUS2yJYeNwL1sSzAKRXLiGUbEaMkGEMUCkKS8-p8CAjA_pvu3ADiMd10ARfW8qyHohU8Klzfo_iirmzz13E48mVhcoHOryn3Sq197SguLJY8=<</CIPHER>>
```
Comment (meliGuzman) [inline ops/terraform/env/mgmt/base_config/values/mgmt.yaml:53]: @dondevun can you also remove adrian.jones@adhocteam.us and kirsten.hansen@adhocteam.us? They were on the ad hoc team on BB2

File: ops/terraform/env/mgmt/base_config/values/mgmt.yaml [hunk 2]
```diff
@@ -56,6 +50,6 @@ bfd:
...
...sAxQd8gM91NwX7Zpgr5jx-zcqVk7fWkXOOhw4wB2VFl0575QyroLOhKerwWtu4GsIIbcL0oLffLRIZk0VLcGTNJCaAjA2hmUMq4kOBcTydPnHFQOM2hpxez5ywscjsR1voCMfC-Iy711wnngD21kg4-1ofHHPJZnCeCcyFS_QNi6Pr4N_uA0nKOMRRqguTaELA6MnLgQ33TS50Q7T2HJR415WAwVv-Wtr4uro5yV2mJ6DMDHBTLwnBecH7GO7EX9O2vQA8RfWIwzoECPwaHjOgSuj299DJ3hf2WtmTT3APw7h5QTaeUdGYyxHtN_AfPJ5TDX76d3tASyIspBwFNRVlm64hoocR8O1Y65pgfOzZyotXrsmVqgpbdbTaGxup9wsmNaQoFXhKvUe-aLfCUG_ToTFy9k9UiV9zgLUbFWLiJbrihb6OMuSWQvI-8LIArONeS7OFq-LkvtsPZo8esgDnKfv7u130LiXGlj90HPYWZe23HdcdnglZg1ZrwTfIl1LFpuJGsWDefJ4KSUX2sZ5A0uuZo2jLDQ5pnKKYpW-LOnzswb2PTBkHOFO-xP0scpNd1osdip23aRfX5hD3GcEWG5v4YSQvhVXq7o59X6CDQC64v30QVK4vYW8dnfnDwqqRttUyQkUn4TN8Z-_dyYYwQHYB9z1eJX_m5rixOiaEas2DKY4JoCnrjnkPjtGOruFSEa3nSmcNHdGuK_752k8hlx6MmKnjhNbJbjOiAcyZzZsRmgfWk3as2hWVuJnmt3n0MJLDwk7cRCGVN1XMOkgTSN6M9AHeq44GaURSB052K7u3pUN1OC35GOcov_s723xdlc8urrJRszcW5IFD_zJK2mI8loqkx_C-HJKFoNxpZQPSIfV0wtPBRt40o_KZxdLV3CCE5VUUOyAHZ0MZnRnAawmkeev3s_36RuTzEVWvzLDEl7RBN2unzX4HGgxMmMn1daGiNZtPUdzf1evDhwFROfhrMB3sCvVlyy-Ncr_gFOk9Qr5m9Rn6CVTzVWzFOv791JwASFuXuO5by1z8H_cPZsAZzBlAjEA5IQhCBaL-lpqKDnVET1Z4oLIWjxjb6TujBYFbewoCCoffQJ-zt0nqtH8dbp20uFBAjBk0rNPFOPF7HnDqwqS9GJjc2rXNv91P7B9UP8_KxjBjUPSumDSn8oKOJ2rdrwvgQs=<</CIPHER>>
```
Comment (meliGuzman) [inline ops/terraform/env/mgmt/base_config/values/mgmt.yaml:59]: @dondevun also you deleted my access, can you bring it back?

File: ops/terraform/env/mgmt/base_config/values/mgmt.yaml (hunk 1 above)
Comment (dondevun) [reply ops/terraform/env/mgmt/base_config/values/mgmt.yaml]: Done!

**Review by mjburling — APPROVED** [review]
Cool. This is going to need a little bit of attention when we go to apply this, particularly against the QuickSight resources, but it's in good shape. Thanks!

**Pull request merged**